api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -600,6 +600,11 @@ public void mergePatients_shouldChangeUserRecordsOfNonPreferredPersonToPreferred
 		executeDataSet(USERS_WHO_ARE_PATIENTS_XML);
 		Patient notPreferred = patientService.getPatient(2);
 		voidOrders(Collections.singleton(notPreferred));
+
+		Calendar calendar = Calendar.getInstance();
+		calendar.set(1950, Calendar.FEBRUARY, 2);
+		patientService.getPatient(6).setBirthdate(calendar.getTime());
```

Comment:
Note that the you're adding extra in the file that matches the value of USERS_WHO_ARE_PATIENTS_XML, may be the patient is in there